space/components/issues/peek-overview/comment/comment-detail-card.tsx

```diff
@@ -90,20 +97,16 @@ export const CommentCard: React.FC<Props> = observer((props) => {
             className={`flex-col gap-2 ${isEditing ? "flex" : "hidden"}`}
           >
             <div>
-              <Controller
-                control={control}
-                name="comment_html"
-                render={({ field: { onChange, value } }) => (
-                  <TipTapEditor
-                    workspaceSlug={workspaceSlug as string}
-                    value={value}
-                    debouncedUpdatesEnabled={false}
-                    customClassName="min-h-[50px] p-3 shadow-sm"
-                    onChange={(comment_json: Object, comment_html: string) => {
-                      onChange(comment_html);
-                    }}
-                  />
-                )}
+              <TipTapEditor
+                workspaceSlug={workspaceSlug as string}
+                ref={editorRef}
+                value={watch("comment_html")}
```

Comment:
Can you replace watch function with the Controller component, just like previous code.